packages/dds/sequence/src/test/intervalCollection.fuzzUtils.ts

```diff
@@ -58,13 +60,20 @@ export interface ChangeProperties extends IntervalCollectionSpec {
 	properties: PropertySet;
 }
 
-export type IntervalOperation = AddInterval | ChangeInterval | DeleteInterval | ChangeProperties;
+export interface RevertSharedStringRevertibles extends IntervalCollectionSpec {
+	type: "revertSharedStringRevertibles";
+	id: string;
```

Comment:
`id` should be unnecessary: the fuzz harness handles selecting a client to perform the operation on since this is a really common concern.

A lot of the interval operations have an "id" field on them because in addition to selecting a client, they need to also select which interval to do something with.